src/pug/_mixins.pug

```diff
@@ -28,7 +28,7 @@
   const redirect = {
     db:            '/db/stats',
     contribute:    '/contribute/open_source',
-    blog:          '/blog/report_18',
+    blog:          '/blog/year_one_recap',
```

Comment:
Can you rename this to "recap_year_1" or "recap_month_12" or some title that can be re-used later.